docs/security.rst

```diff
@@ -35,7 +35,7 @@ The general process is as follows:
 1. Receive a private report of a security issue
 2. Acknowledge receipt of the report
 3. Post advance notice to the GitHub repo indicating that a security issue exists
-4. Prepare a [GitHub Security Advisory](https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/about-repository-security-advisories)
+4. Prepare a `GitHub Security Advisory`_
```

Comment:
Synax: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages